bookstore/bookstore_config.py

```diff
@@ -36,3 +36,12 @@ class BookstoreSettings(LoggingConfigurable):
         16,
         help="Maximum number of threads for the threadpool allocated for S3 read/writes",
     ).tag(config=True)
+
+
+def validate_bookstore(settings: BookstoreSettings):
```

Comment:
Let's go ahead and add a simple docstring for the autodocs. Examples are in the papermill docs PR that is soon to merge. numpy format.

Reply:
Great idea!